twisted/names/server.py

```diff
@@ -70,22 +70,22 @@ class DNSServerFactory(protocol.ServerFactory):
     def __init__(self, authorities=None, caches=None, clients=None, verbose=0):
         """
         @param authorities: Resolvers which provide authoritative answers.
-        @type authorities: L{list} of L{IResolver} providers
+        @type authorities: C{list} of C{IResolver} providers
 
         @param caches: Resolvers which provide cached non-authoritative
             answers. The first cache instance is assigned to
             C{DNSServerFactory.cache} and its C{cacheResult} method will be
             called when a response is received from one of C{clients}.
-        @type caches: L{list} of L{Cache<twisted.names.cache.Cache} instances
+        @type caches: C{list} of C{Cache<twisted.names.cache.Cache} instances
```

Comment:
No.  However, there is a problem on this line, but you didn't fix it.  It should be:

@type caches: L{list} of L{Cache<twisted.names.cache.Cache>} instances